nixos/tests/age-plugin-tpm-decrypt.nix

```diff
@@ -0,0 +1,33 @@
+import ./make-test-python.nix (
+  { pkgs, ... }:
+  {
+    name = "age-plugin-tpm-decrypt";
+    meta = with pkgs.lib.maintainers; {
+      maintainers = [ sgo ];
+    };
+
+    nodes.machine =
+      { pkgs, ... }:
+      {
+        virtualisation.tpm.enable = true;
+        environment.systemPackages = with pkgs; [
+          age
+          age-plugin-tpm
+        ];
+      };
+
+    testScript = ''
+      machine.start()
+      machine.wait_for_unit("multi-user.target")
```

Comment:
Do we need that?

Reply:
```suggestion
```
I tested it locally and it works without it